ert_shared/storage/extraction_api.py

```diff
@@ -34,12 +35,15 @@ def _extract_and_dump_observations(rdb_api, blob_api):
         facade.get_observation_key(nr) for nr, _ in enumerate(facade.get_observations())
     ]
 
-    measured_data = MeasuredData(facade, observation_keys)
+    if len(observation_keys) > 0:
+        measured_data = MeasuredData(facade, observation_keys)
```

Comment:
```suggestion
    if len(observation_keys) == 0:
        return    
    measured_data = MeasuredData(facade, observation_keys)
```

Reply:
Matter of preference, but might be better suited as it gives us more line space